### PR TITLE
Add Appwrite CLI local dev directory to .gitignore files

### PR DESCRIPTION
**Pull request by ebenezerdon**
## What does this PR do?

Adds Appwrite CLI local dev directory (.appwrite) to templates/runtimes .gitignore files

## Related PRs and Issues
x

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -129,4 +129,7 @@ dist
 .yarn/install-state.gz
 .pnp.*
 
-appwrite.json
\ No newline at end of file
+appwrite.json
+
+# VSCode history directory
+.history
```
Comment (Meldiron) [inline .gitignore:135]: I found 76 gitignore files with `.appwrite`:

CleanShot 2024-07-09 at 17 19 26@2x


But I found 78 files with at least 1 character in `.gitignore`:

CleanShot 2024-07-09 at 17 19 19@2x


I think that means we missed 2 gitignore files

File: .gitignore
Comment (Meldiron) [inline .gitignore:1]: Some tempaltes ar emissing gitignore all-together. Lets add them - you can copy gitignore file from a different template of the same runtime.

- `bun/whatsapp-with-vonage` - copy from `bun/starter`
- `go/starter` - it doesn't have .gitignore at all. Let's add it, and only mention .appwrite there
- `ruby/sync_with_meilisearch` - Lets copy from `ruby/starter`
- node/query-mongo-atlas - missing gitignore all together. copy from node/starter
- node/query-neo4j-auradb - missing gitignore all together. copy from node/starter
- node/query-neon-postgres - missing gitignore all together. copy from node/starter
- node/query-redis-labs - missing gitignore all together. copy from node/starter
- node/query-upstash-vector - missing gitignore all together. copy from node/starter
- node/speak-with-elevenlabs - missing gitignore all together. copy from node/starter 
- node/speak-with-lmnt - missing gitignore all together. copy from node/starter

File: .gitignore (hunk 1 above)
Comment (Meldiron) [reply .gitignore:135]: Nevermind. One is root level, and one is CI/CD action. You got them all

File: .gitignore
Comment (ebenezerdon) [reply .gitignore:1]: Done. Please review new changes

**Review by Meldiron — APPROVED** [review]

**Pull request merged**